### DISCUSSION TITLE
The json is seen for a few seconds before its content is displayed.

### DISCUSSION THREAD

**Discussion by Guille2307** [ocombe/ng2-translate#1482] [General]
Hello colleagues, I have a problem. The json is seen for a few seconds before its content is displayed.

How can I prevent those seconds from being displayed? 
I am using Angular 15

I appreciate your help
![image (1)](https://github.com/ngx-translate/core/assets/98533944/ff47a8d2-8197-4bdf-9b81-c6fd0feb698a)

**Comment by emancol** [comment]
Hello. You should give us how is structured your Module,  if you are making an API Service and so.

**Comment by CodeAndWeb** [comment]
There are practically 2 ways to solve this:

1) Include and set your default language in the code. E.g. by importing the JSON files directly -> `setTranslation()`
2) Display a kind of loading screen until the observable returned by `use()` resolves.
